lib/parseTriggersEmulator.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var FirebaseError = require('./error');
+var fork = require('child_process').fork;
+var path = require('path');
+var RSVP = require('rsvp');
+
+var TRIGGER_PARSER = path.resolve(__dirname, './triggerParser.js');
+
+module.exports = function(firebaseConfig, projectId, dbInstance, sourceDir) {
+  return new RSVP.Promise(function(resolve, reject) {
+    var env = {
+      GCLOUD_PROJECT: projectId,
+      DB_NAMESPACE: dbInstance,
```

Comment:
GCLOUD_PROJECT currently already exists in the GCF runtime. It is used by each provider when constructing the builder, eg: https://github.com/firebase/firebase-functions/blob/master/src/providers/auth.ts#L31